tests/llm/fixtures/test_ask_holmes/64_keda_vs_hpa_confusion/test_case.yaml

```diff
@@ -0,0 +1,13 @@
+user_prompt: 'Why isnt KEDA scaling the invoice-generator deployment?'
+description: Verify Holmes goes beyond 'technically correct' answers and provides a useful answer based on the user's actual intent
```

Comment:
is this new?

Reply:
yeah its just me documenting what we're trying to accomplish here in a human readable format. not used for anything